app/battle/start.py

```diff
@@ -0,0 +1,16 @@
+from app.knight.creation import Knight
+
+
+def check_hp(knights: list[Knight]) -> list[Knight]:
+    for knight in knights:
+        if knight.hp <= 0:
+            knight.hp = 0
+
+    return knights
+
+
+def get_damage(knight1: Knight, knight2: Knight) -> list[Knight]:

```

Comment:
The function `get_damage` is correctly defined with type annotations. Ensure that the knights passed to this function are instances of the `Knight` class.